src/MyAnimeList/MyAnimeList/Domain/Studio.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace MyAnimeList.Domain;
+
+public class Studio
```

Comment:
O mapping do Studio falta as prop de navegação, dito isso com a base de dados que nós temos estamos repetindo o studio, com a base que eu tenho é uma diff de 11.295 para 721, com a base real esse número vai ser bem pior
o ideial é fazer um many to many aqui

Reply:
Em progresso.  Estou criando a tabela intermediária para a relação Many to Many, e terei que mudar a consulta da Issue para trabalhar com esta nova configuração.